recipes/fastjet/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "fastjet" %}
+{% set version = "3.3.4.0rc8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 30ffebf47389c7e2d8b40a6e863175848fe59c8afc5e276612b7ccf3b2a33ed5
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - gmp
+  host:
+    - pip
+    - pybind11
+    - python
+  run:
+    - awkward >=1
+    - numpy >=1.13.3
+    - python
+    - vector
+
+test:
+  imports:
+    - fastjet
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/scikit-hep/fastjet
+  summary: Official FastJet bindings to Python and Awkward Array.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - lgray
+    - jpivarski
```

Comment:
@jpivarski Please 👍 to being a maintainer

Reply:
Yes, I +1 to being a maintainer.